package_creator.py

```diff
@@ -82,6 +88,30 @@ def insert_image_to_yml(dir_name, package_path, yml_data, yml_text):
     return yml_text, found_img_path
 
 
+def insert_description_to_yml(dir_name, package_path, yml_data, yml_text):
```

Comment:
lets add:
if dir_name != "Integrations":
....

And remove it from line 107

Reply:
Ah, I forgot to dedup that